containers/repronim-reprostim/generate_container_ci.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -eu
+
+thisdir=$(dirname "$0")
+
+PYTHON_VERSION=3.10
+
+PSYCHOPY_VERSION=2024.2.5
+PSYCHOPY_INSTALL_DIR=/opt/psychopy
+PSYCHOPY_HOME=${PSYCHOPY_INSTALL_DIR}/psychopy_${PSYCHOPY_VERSION}_py${PYTHON_VERSION}
+
+REPROSTIM_VERSION=$(git describe --tags --abbrev=0 2>/dev/null || echo "0.0.1")
+# Decided to go without version to make diff easier to analyze etc
+REPROSTIM_SUFFIX=repronim-reprostim # -${REPROSTIM_VERSION}
+REPROSTIM_GIT_HOME=$(git rev-parse --show-toplevel)
+REPROSTIM_HOME=/opt/reprostim
+
+echo "REPROSTIM_GIT_HOME: ${REPROSTIM_GIT_HOME}"
+echo "REPROSTIM_HOME: ${REPROSTIM_HOME}"
+
+generate() {
+	[ "$1" == singularity ] && add_entry=' "$@"' || add_entry=''
```

Comment:
please avoid generating largely duplicated scripts, parametrize them. Eg. crude parametrization for deploying local development copy we did in the https://github.com/ReproNim/reproin/blob/HEAD/generate_container.sh